.github/workflows/test-docs.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow sets the test / pack and test / tests status check to success in case it's a docs only PR and test-linux.yml is not triggered
```

Comment:
```suggestion
# This workflow sets the 'test-linux / pack' and 'test-linux / tests' status check to success in case it's a docs only PR and test-linux.yml is not triggered
```